test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp

```diff
@@ -63,6 +63,36 @@ uint64_t clz(uint64_t _v)
 	return r;
 }
 
+/// Count trailing zeros for uint64
+uint64_t ctz(uint64_t _v)
+{
+	uint64_t r = 0;
+	unsigned count = 64;
+	while (count-- && (_v & 1))
+	{
+		r++;
+		_v >>= 1;
+	}
+	return r;
+}
+
+/// Count number of bits set for uint64
+uint64_t popcnt(uint64_t _v)
+{
+	// This is only an optimisation.
+	if (_v == 0)
+		return 0;
```

Comment:
Is running it on zeroes known to be common enough to be worth this optimizaation?

Reply:
We can remove it.